pkg/controllers/resources/pods/translate.go

```diff
@@ -52,6 +53,24 @@ func (s *podSyncer) getK8sIPDNSIPServiceList(ctx *synccontext.SyncContext, vPod
 }
 
 func (s *podSyncer) translateUpdate(pObj, vObj *corev1.Pod) (*corev1.Pod, error) {
+	ctrlRuntimeClient, err := client.New(s.physicalClusterConfig, client.Options{})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Is this intentionally a non-cached client?
If not, then we should just pass the `ctx.PhysicalClient` to this function from the `(s *podSyncer) Sync` where it gets called.

Reply:
my bad, I only looked at `podSyncer.physicalClient` which was a `kubernetes.Interface`. I didn't realize we had a `client.Client` from `context`